api/src/pcapi/routes/native/v1/feedbacks.py

```diff
@@ -0,0 +1,43 @@
+import logging
+
+from sqlalchemy.orm import joinedload
+
+from pcapi.core.bookings import models as bookings_models
+from pcapi.core.offers import models as offers_models
+from pcapi.core.users import models as users_models
+from pcapi.core.users.young_status import young_status
+from pcapi.routes.native.security import authenticated_and_active_user_required
+from pcapi.serialization.decorator import spectree_serialize
+
+from .. import blueprint
+from .serialization import feedbacks as serializers
+
+
+logger = logging.getLogger(__name__)
+
+
+@blueprint.native_route("/feedback", methods=["POST"])
+@spectree_serialize(on_success_status=204, api=blueprint.api)
+@authenticated_and_active_user_required
+def post_feedback(user: users_models.User, body: serializers.PostFeedbackBody) -> None:
+    not_cancelled_bookings_query = bookings_models.Booking.query.options(
+        joinedload(bookings_models.Booking.stock)
+        .joinedload(offers_models.Stock.offer)
+        .load_only(offers_models.Offer.id)
+    ).filter(
+        bookings_models.Booking.userId == user.id,
+        bookings_models.Booking.status != bookings_models.BookingStatus.CANCELLED,
+    )
+
+    logger.info(
+        "User feedback",
+        extra={
+            "feedback": body.feedback,
+            "age": user.age,
+            "status": young_status(user).status_type.value,  # type: ignore[attr-defined]
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏mypy a raison ici non ?
Si tu as un user pour lequel le status est None, ça va 500
Même si on ne montre le formulaire qu'à une partie des users, ça coute pas cher d'être un peu défensif ici